manage-apps/register.html

```diff
@@ -206,6 +206,10 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         error: function(res) {
+          console.log('******',res)
+          if (res.statusText === 'error') {
```

Comment:
You're very right. I didn't encounter any double errors but that certainly doesn't mean they couldn't happen! I've committed a tweak that should help, but I haven't been able to get an error yet in my local test to make sure - I'll let you know when I do.